packages/openapi-generator/src/document-converter.ts

```diff
@@ -61,7 +61,7 @@ export async function convertDocToOpenApiV3(
     return (await convert(openApiDocument, {})).openapi;
   } catch (err) {
     throw new ErrorWithCause(
-      'Could not convert OpenAPI specification to OpenAPI version 3.',
+      `Could not convert OpenAPI specification to OpenAPI version 3. ${err.message}`,
```

Comment:
Seems a bit unclean to me, because when using this programatically, you would get both errors. Is there a way to configure how we log the errors in the CLI?

Reply:
Some bullet points after the discussion:
- Just like the upcoming discussion with CAP, we need to consider different audience.
    - CLI only uses the `message`, while programming call  uses the whole object `ErrorWithCause`
- For different audiences, we might build error to different format
    - A single error message
    - Like the stack object structure with hierarchy
    - Nicer and shorter error message that contains all useful information from stack.

Before the CAP meeting, I would keep a smaller change, as this brings the value (real reason) to the CLI user.

Reply:
@marikaner , please approve.